test/service/test_azul_webservice.py

```diff
@@ -0,0 +1,221 @@
+import os
+import sys
+import logging
+import requests
+from threading import Thread
+from unittest import TestCase
+from urllib.parse import urljoin
+from chalice.local import LocalDevServer
+from chalice.config import Config
+
+sys.path.append(os.path.join(os.environ['AZUL_HOME'], 'lambdas', 'service'))
```

Comment:
Could you wrap this environment variable with a property in `azul.Config` and use that method here instead? Search for places where AZUL_HOME is used in Python code and replace those with the new property, too.

Reply:
I searched the Azul codebase and test_azul_webservice.py is the only python file to use the AZUL_HOME environment variable.